tests/php/Extension/FluentDirectorExtensionTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace TractorCow\Fluent\Tests\Extension;
+
+use Page;
+use SilverStripe\CMS\Model\SiteTree;
+use SilverStripe\Control\Director;
+use SilverStripe\Dev\FunctionalTest;
+use TractorCow\Fluent\Extension\FluentDirectorExtension;
+use TractorCow\Fluent\Extension\FluentFilteredExtension;
+use TractorCow\Fluent\Extension\FluentSiteTreeExtension;
+use TractorCow\Fluent\State\FluentState;
+
+/**
+ * Class FluentDirectorExtensionTest
+ *
+ * @package TractorCow\Fluent\Tests\Extension
+ */
+class FluentDirectorExtensionTest extends FunctionalTest
+{
+    /**
+     * @var string
+     */
+    protected static $fixture_file = 'FluentDirectorExtensionTest.yml';
+
+    /**
+     * @var array
+     */
+    protected static $required_extensions = [
+        SiteTree::class => [
+            FluentSiteTreeExtension::class,
+            FluentFilteredExtension::class,
+        ],
+        Director::class => [
+            FluentDirectorExtension::class,
+        ],
+    ];
+
+    public function setUp() // phpcs:ignore SlevomatCodingStandard.TypeHints
```

Comment:
setUp should be protected in PHPUnit 5.x

Reply:
This should have been fixed :(

Reply:
oops, I missed that X_X